src/spatialdata/__init__.py

```diff
@@ -1,5 +1,17 @@
 from __future__ import annotations
 
+import dask
+
+dask.config.set({"dataframe.query-planning": False})
```

Comment:
I don't get it, it's set to false here, but then once dask.dataframe is imported, is set again to True?

Reply:
It's not obvious, I'll add a comment to make it clear. What can happen is that the user imports `dask.dataframe` before importing this file/setting `dataframe.query-planning` to `False`. In that case `DASK_EXPR_ENABLED` would be `True`, but we don't want that. So we add this extra check.